scripts/adopters/index.js

```diff
@@ -0,0 +1,25 @@
+const { readdir, writeFile, readFile } = require('fs').promises;
+const { convertToJson } = require('../utils');
+const { resolve } = require('path');
+
+const dirWithAdopters = 'config/adopters';
+module.exports = async function buildAdoptersList() {
+    let files = await readdir(dirWithAdopters);
```

Comment:
adopters.yml is just a single file, and will be like this long term, so why we need code that reads all files from adopters dir. Maybe just remove adopters dir and add adopters.yml directory to config, and just read that one single file and transform to json